### PR TITLE
Rename, move files, emove uncessary file

### PR DESCRIPTION
**Pull request by huyngse**

### CODE REVIEW

File: src/pages/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from "react";
+import ListCart from "../components/CageList/CageList.jsx";
+import BackToTopButton from "../components/BackToHome.jsx";
+import { products } from "../components/CageList/list_product.jsx";
+import { useCart } from "../contexts/CardContext.jsx";
+import "bootstrap/dist/css/bootstrap.css";
+import "bootstrap/dist/js/bootstrap.js";
+import "bootstrap-icons/font/bootstrap-icons.css";
+import HeaderMenu from "../components/HeaderMenu.jsx";
+import Carousel from "../components/Carousel.jsx"
```
Comment (Cuong3103) [inline src/pages/HomePage.jsx]:
```suggestion
import React, { useState, useEffect } from "react";
import "bootstrap/dist/css/bootstrap.css";
import "bootstrap/dist/js/bootstrap.js";
import "bootstrap-icons/font/bootstrap-icons.css";

import ListCart from "../Components/CageList/CageList.jsx";
import BackToTopButton from "../Components/BackToHome.jsx";
import { products } from "../Components/CageList/list_product.jsx";
import { useCart } from "../contexts/CardContext.jsx";
import HeaderMenu from "../Components/HeaderMenu.jsx";
import Carousel from "../Components/Carousel.jsx"
```

File: src/pages/LoginPage.jsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import Cookies from "js-cookie";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
```
Comment (Cuong3103) [inline src/pages/LoginPage.jsx]:
```suggestion
import axios from "axios";

import { useNavigate } from "react-router-dom";
```

File: src/pages/LoginPage.jsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+import { useAuth } from "../contexts/AuthContext.jsx";
+import { LoginInput } from "../components/LoginInput.jsx";
```
Comment (Cuong3103) [inline src/pages/LoginPage.jsx:7]:
```suggestion
import { LoginInput } from "../Components/LoginInput.jsx";
```

File: src/pages/ProductDetailPage.jsx [hunk 4]
```diff
@@ -0,0 +1,337 @@
+import React, { useContext } from "react";
+import { useParams } from "react-router-dom";
+import { products } from "../components/CageList/list_product";
+import Rating from "@mui/material/Rating";
+import AddShoppingCartIcon from "@mui/icons-material/AddShoppingCart";
+import { CardContext } from "../contexts/CardContext.jsx";
+import ImageGallery from "react-image-gallery";
+import QuantityInput from "../components/QuantityInput";
+import UserReview from "../components/UserReview";
+import MainLayout from "../layouts/MainLayout";
```
Comment (Cuong3103) [inline src/pages/ProductDetailPage.jsx]:
```suggestion
import { useParams } from "react-router-dom";
import Rating from "@mui/material/Rating";
import AddShoppingCartIcon from "@mui/icons-material/AddShoppingCart";
import ImageGallery from "react-image-gallery";

import { CardContext } from "../contexts/CardContext.jsx";
import { products } from "../Components/CageList/list_product.jsx";
import QuantityInput from "../Components/QuantityInpu.jsx";
import UserReview from "../Components/UserReview";
import MainLayout from "../layouts/MainLayout";
```

File: src/pages/SignUpPage.jsx [hunk 5]
```diff
@@ -0,0 +1,120 @@
+import * as React from "react";
+import Grid from "@mui/material/Grid";
+import { styled } from "@mui/material/styles";
+import { CustomInput } from "../components/LoginInput.jsx";
```
Comment (Cuong3103) [inline src/pages/SignUpPage.jsx]:
```suggestion
import { styled } from "@mui/material/styles";

import { CustomInput } from "../Components/LoginInput.jsx";
```

**Review by Cuong3103 — CHANGES_REQUESTED** [review]

**Review by huyngse — COMMENTED** [review]
Fixed

**Review by Cuong3103 — APPROVED** [review]

**Pull request merged**